tsconfig.json

```diff
@@ -4,7 +4,7 @@
 
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es2020",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Any particular reason for this change?

Reply:
I thought I had a bug with optional chaining, but it turns out my WSL node was just outdated for some reason (actually, for some reason the entire thing was targeting Node 12, I've upped that to Node 14). It's back to ES2020.